dataland-frontend/src/components/general/ModalsComponent.vue

```diff
@@ -0,0 +1,102 @@
+<template>
+  <template v-if="_isModal()">
+    <component
+      :is="_findModalComponent().triggerComponent"
```

Comment:
Why the name **trigger**Component?

Reply:
Yes. We need to define two components. One that will render some kind of link or button (triggerComponent) and it opens the modal. The other component is which component is to be displayed within the modal.

Reply:
Yup, wrote this comment to early. `triggerComponent` is fine. I just want to suggest `modalTriggerComponent` for more clearity, but I let this decision to you.